small-talk/db/hcTest.js

```diff
@@ -0,0 +1,19 @@
+"use server"
+const uri = "mongodb+srv://smt_root:pokemonwithguns@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
+import Mongoboi from "./mongo"
+export default async function handleCreate(username) {
+    "use server"
+    const mongoboi = new Mongoboi(uri, "Users")
+    const date = new Date('January 1, 1980');
```

Comment:
Is a date needed somewhere? I see this here as well as in hsTest but not sure if they're being used.

Reply:
Date should be used to save dob in unix time

Reply:
So this is something that just needs to be added in?

Reply:
we can make sure that they are being used in the test coverage